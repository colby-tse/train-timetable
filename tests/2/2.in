Coffs Harbour::Koolewong::20:00:00
Coffs Harbour::Koolewong::19:48:30
Coffs Harbour::Koolewong::21:20:23
Coffs Harbour::Koolewong::22:10:01
