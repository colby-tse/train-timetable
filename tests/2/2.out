The next train to Koolewong from Coffs Harbour departs at 19:48:30
